examples/democoin/x/assoc/validator_set_test.go

```diff
@@ -53,19 +53,23 @@ func TestValidatorSet(t *testing.T) {
 	require.Equal(t, base.Validator(ctx, addr1), valset.Validator(ctx, addr1))
 	require.Equal(t, base.Validator(ctx, addr2), valset.Validator(ctx, addr2))
 
-	assocs := valset.Associations(ctx, addr1)
-	require.Equal(t, 1, len(assocs))
-	require.True(t, bytes.Equal(assoc1, assocs[0]))
+	// XXX: Will be fixed by #2248
```

Comment:
TODO is there an issue?

Reply:
Added comment to 2248

Reply:
I fixed it in this PR - turns out that the prefixing implementation was wrong